_sass/components/_footer.scss

```diff
@@ -0,0 +1,154 @@
+.c-footer {
+  background-color: $dark-blue;
+  width: 100%;
+
+  &__container {
+    width: 80%;
+    margin: 0 auto;
+    padding: 4rem 0;
+  }
+
+  &__award-container {
+    width: 100%;
+    padding: 4rem 0;
+  }
+
+  &__award-image {
+    width: 5rem;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__award-text {
+    @include uppercase-white-font;
+    text-align: center;
+    margin: 1.5rem auto 0;
+    max-width: 23.625rem;
+  }
+
+  &__link-lists {
+    @include link-lists-flex;
+  }
+
+  &__link-list {
+    width: 100%;
+    padding: 0 0.625rem 0 0;
+    box-sizing: border-box;
+    margin: 1.25rem 1.25rem 0;
+
+    @media (min-width: $bp-footer-sm) {
+      width: 45%;
+      margin: 1.25rem 0.625rem 0;
+    }
+
+    @media (min-width: $bp-lg) {
+      width: 20%;
+      margin: 1.25rem 0.625rem;
+    }
+
+    @media (min-width: $bp-xl) {
+      width: 14%;
+      margin: 1.25rem 0.625rem 40px;
+    }
+  }
+
+  &__link-title {
+    @include uppercase-white-font;
+    margin: 0 0 1.25rem;
+
+  }
+
+  &__link {
+    color: $medium-blue;
+    font-size: 1rem;
+    line-height: 1.5;
+    text-decoration: none;
+
+    &--underlined {
+      color: $white;
+      text-decoration: underline;
+    }
+  }
+
+  &__list-item {
+    margin-bottom: 1.25rem;
+  }
+
+  &__social-list {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0 0.625rem 0 0;
+    box-sizing: border-box;
+    padding: 2rem 0;
+    margin: 0 auto;
+  }
+
+  &__social-item {
+    & + & {
+      margin-left: 1rem;
+    }
+  }
+
+  &__copyright-container {
+    width: 100%;
+    padding: 0 0.625rem 0 0;
+    box-sizing: border-box;
+    margin: 2.5rem 1.25rem 0.625rem;
+
+    @media (min-width: $bp-footer-sm) {
+      width: 45%;
+        }
+
+    @media (min-width: $bp-lg) {
+      width: 20%;
+      margin: 1.25rem 0.625rem 0.625rem;
+    }
+
+    @media (min-width: $bp-xl) {
+      width: 14%;
+      margin: 1.25rem 0.625rem 2.5rem;
+    }
+  }
+
+  &__copyright {
+    color: $white;
+    line-height: 1.5;
+    max-width: 17.5rem;
```

Comment:
I think this max-width could be removed since there is a max-width on the surrounding container. There won't be a line break between the two sentences at each screen size, but that's okay. I've learned that it's better to let the design be flexible and let the content flow rather than trying to control it too much.

Reply:
makes sense 👌